### ISSUE TITLE
WBN: show positional profiles on gcode visualizer

### ISSUE THREAD

**Issue by cabbey** [FormerLurker/Octolapse#81]
I don't know if this is even possible with the way octoprint's plugins work. But if it is, it would be very nice to be able to overlay the snapshot restrictions geometry on the gcode viewer so folks can see if their print will or will not cross into the right area.

**Comment by FormerLurker** [comment]
Yes, I have thought about this.  I'll have to do some digging, but that would be really cool!  Marking as an enhancement.
